landingai/pipeline/frameset.py

```diff
@@ -380,6 +433,67 @@ def downsize(
             frame.downsize(width, height)
         return self
 
+    def crop(self, bbox: Tuple[int, int, int, int]) -> "FrameSet":
+        """Crop the images based on the bounding box
+
+        Parameters
+        ----------
+        bbox: A tuple with the bounding box coordinates (xmin, ymin, xmax, ymax)
+        """
+        for frame in self.frames:
+            frame.crop(bbox)
+        return self
+
+    def adjust_sharpness(self, factor: float) -> "FrameSet":
+        """Adjust the sharpness of the image
+
+        Parameters
+        ----------
+        factor: The enhancement factor
+        """
+        for f in self.frames:
+            f.adjust_sharpness(factor)
+        return self
+
+    def adjust_brightness(self, factor: float) -> "FrameSet":
+        """Adjust the brightness of the image
+
+        Parameters
+        ----------
+        factor: The enhancement factor
+        """
+        for f in self.frames:
+            f.adjust_brightness(factor)
+        return self
+
+    def adjust_contrast(self, factor: float) -> "FrameSet":
+        """Adjust the contrast of the image
+
+        Parameters
+        ----------
+        factor: The enhancement factor
+        """
+        for f in self.frames:
+            f.adjust_contrast(factor)
+        return self
+
+    def adjust_color(self, factor: float) -> "FrameSet":
+        """Adjust the color of the image
+
+        Parameters
+        ----------
+        factor: The enhancement factor
+        """
+        for f in self.frames:
+            f.adjust_color(factor)
+        return self
```

Comment:
Isn't there some cool pattern in Python where we can annotate methods like this that are a pass through and have the language deal with it??

Reply:
There are a few ways to do it... but it would just save us 1 line of code (specially considering that we still need the documentation comment for docs), and it would make the code slightely less readable. I'm not sure it worth the effort in this case...